src/bindings/swig/python/plugins/dns/README.md

```diff
@@ -0,0 +1,31 @@
+- infos = Information about the python plugin is in keys below
+- infos/author = Lukas Hartl <git@lukashartl.at>, Leonard Guelmino <e1503940@student.tuwien.ac.at>
+- infos/licence = BSD
+- infos/provides = check
+- infos/status = maintained
+
+## DNS Python plugin
+
+This filter plugin checks if a Key is a valid domain name, i.e. if the name can be resolved into an IPv4 address.
+The validation takes place whenever the `kdb set` method is called.
+
+## Usage
+
+The python plugin requires the configuration parameter **script** holding the file path to the
+python script. The mount command looks like
+
+```sh
+sudo kdb mount file.ini /python python script=/path/to/dns_plugin.py
+```
+
+For the plugin to actually check a certain key, the `check/dns` metakey must be set.
```

Comment:
Afaics the functionality of your plugin is identical to https://www.libelektra.org/plugins/network

If this is the case, you should also use the same metadata: check/ipaddr that accept the values ipv4 or ipv6

If you want to make your plugin different, what about checking CIDR-notation/subnet/masking?